### DISCUSSION TITLE
How to implement a chat interface, ChatGPT that has become popular recently

### DISCUSSION THREAD

**Discussion by yuanjie-ai** [zauberzeug/nicegui#835] [Q&A] [answered]
### Description

怎么实现一个聊天界面，最近大火的chatgpt

**Comment by rodja** [comment]
Translation: 
>How to implement a chat interface, chatgpt, which has become popular recently

**Comment by rodja** [comment, marked as answer]
Have a look at our [chat app example](https://github.com/zauberzeug/nicegui/blob/main/examples/chat_app/main.py). To connect with ChatGPT you will need a Python wrapper like [RevChatGPT](https://github.com/acheong08/ChatGPT) or [chatgpt-wrapper](https://github.com/mmabrouk/chatgpt-wrapper).
